lib/global_config.js

```diff
@@ -157,3 +158,44 @@ const rest = {
   HTTPS_PROTOCOL: 'https'
 };
 exports.rest = rest;
+
+const defaultXmlParserConfiguration = {
+  ignoreAttributes: true,
+  alwaysCreateTextNode: false,
+  attributeNamePrefix: '@',
+  attributesGroupName: null
+};
+
+/**
+ * Return default configuration for XmlParser.
```

Comment:
do we want to keep it named that it's XmlParser config? maybe it should be more agnostic what library we are using under the hood?

Reply:
I think it is ok. Now we use the same names like fast-xml-parser but it could be understable for others or if we need change the library we can map this.